src/main/java/org/jabref/logic/importer/fetcher/GrobidCitationFetcher.java

```diff
@@ -67,7 +74,7 @@ public List<BibEntry> performSearch(String searchQuery) throws FetcherException
             collect = Arrays.stream(searchQuery.split("\\r\\r+|\\n\\n+|\\r\\n(\\r\\n)+"))
                             .map(String::trim)
                             .filter(str -> !str.isBlank())
-                            .map(this::parseUsingGrobid)
+                            .map(FauxPas.throwingFunction(this::parseUsingGrobid))
                             .flatMap(Optional::stream)
                             .collect(Collectors.toList());
         } catch (RuntimeException e) {
```

Comment:
Is this still necessary? I think, this block can be removed!

Reply:
@koppor I have removed the try/catch block checking for runtime exceptions since we are just using FetcherExceptions now